examples/go/tagging/handler/handler.go

```diff
@@ -94,35 +92,34 @@ func Handle(req handler.Request) (handler.Response, error) {
 		moRef.Value,
 		vcc.Tag.Urn,
 	)
-	log.Println(string(message))
+	log.Println(message)
 
 	err = clients.Govmomi.Logout(ctx)
 	if err != nil {
-		log.Printf("Unable to log out of govmomi due to %v.", err)
+		log.Printf("unable to log out of govmomi due to %v", err)
```

Comment:
usually formatted error handling follows this pattern: `(what happend: %v, err)`